### PR TITLE
Move GPU projects to the GPU pool

### PR DESCRIPTION
**Pull request by mindprince**
- Move GPU projects to the GPU pool and use them.
- Use the new project for GKE PR jobs.

### CODE REVIEW

**Comment by krzyzacy** [comment]
I think we still need to manually enable the apis?

File: boskos/resources.json [hunk 1]
```diff
@@ -455,6 +455,12 @@
       "k8s-gke-gpu-boskos-03",
       "k8s-gke-gpu-boskos-04",
       "k8s-gke-gpu-boskos-05"
```
Comment (krzyzacy) [inline boskos/resources.json]: comma in the end

File: boskos/resources.json [hunk 2]
```diff
@@ -455,6 +455,12 @@
...
+      "k8s-jkns-e2e-gce-plugin-gpus",
+      "k8s-jkns-e2e-gce-gpus",
+      "k8s-jkns-e2e-gce-gpus-1-7",
+      "k8s-jkns-e2e-gke-plugin-gpus",
+      "k8s-jkns-e2e-gke-gpus",
+      "k8s-jkns-e2e-gke-gpus-1-7",
```
Comment (krzyzacy) [inline boskos/resources.json]: and no comma here - world of json :-(

**Comment by krzyzacy** [comment]
new OKR - convert `*.json` to `*.yaml`

File: boskos/resources.json (hunk 1 above)
Comment (mindprince) [reply boskos/resources.json]: Done.

File: boskos/resources.json (hunk 2 above)
Comment (mindprince) [reply boskos/resources.json]: Done.

**Comment by mindprince** [comment]
> I think we still need to manually enable the gke apis?
I did that for these projects.

**Pull request merged**